PowerValidatedSolutions.psm1

```diff
@@ -51303,7 +51303,6 @@ Function Get-AriaNetworksDataSource {
     }
 }
 Export-ModuleMember -Function Get-AriaNetworksDataSource
-
 Function Remove-AriaNetworksDataSource {
```

Comment:
```suggestion

Function Remove-AriaNetworksDataSource {
```

Reply:
Resolved in https://github.com/vmware/power-validated-solutions-for-cloud-foundation/commit/954c74bc51b1fef11dc26b704cc60d52dd1bba80